src/components/global-styles.tsx

```diff
@@ -64,6 +79,22 @@ const SizeStyles = css`
       min-width: 440px !important;
       overflow-x: hidden;
     }
+
+    .intro-card--item {
+      flex-basis: 50%;
+    }
+
+    .home--action-buttons {
+      flex-direction: row;
+    }
+
+    .margin-bottom-bigger {
```

Comment:
@aulneau @kyranjamie @fbwoolf 
I get some automatic styling in some components like
<style data-emotion="css" data-s="">.css-dyo91b>*:not(style):not(:last-child){margin-bottom:8px;}</style>

So I temporarily used !important. I could not find where this comes from yet...
If someone has any idea let me know.

Reply:
I was hoping to get mine merged before more work was done on onboarding bc I restructured some things to make scaling the CSS a bit easier. We try to get my PR merged asap.

Reply:
`*:not(style):not(:last-child){margin-bottom:8px;}` comes from `<Stack>`. Wherever your component is, it's under a Stack and is inheriting those styles. If you do not want it, you must remove it from being a child.

Reply:
@fbwoolf Yes I am planning to merge it after your PR so we can use your new css split.

Reply:
@aulneau Thanks I will check it out.